openadapt/models.py

```diff
@@ -526,7 +545,16 @@ def get_active_window_event(
         Returns:
             (WindowEvent) the active window event.
         """
-        return WindowEvent(**window.get_active_window_data(include_window_data))
+        window_event_data = window.get_active_window_data(include_window_data)
+
+        a11y_event_data = window_event_data.get("state", {}).copy()
```

Comment:
Why `copy()`? Can you please clarify in a comment?

Reply:
It should work without copy too, I'll remove it

Reply:
removed in 
31c270f